### PR TITLE
APIGOV-24088 - remove version file

### PR DESCRIPTION
**Pull request by dfeldick**

### CODE REVIEW

File: .github/workflows/build-apigee-discovery-agent-docker.yml [hunk 1]
```diff
@@ -16,10 +16,11 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Create commit_id file
+      - name: Create  & version file
```
Comment (jcollins-axway) [inline .github/workflows/build-apigee-discovery-agent-docker.yml:19]: commit_id & version

File: .github/workflows/build-apigee-discovery-agent-docker.yml [hunk 2]
```diff
@@ -16,10 +16,11 @@ jobs:
...
         working-directory: discovery
         run: |
           git branch --show-current > commit_id
+          echo "${{ github.ref }}" > version"
```
Comment (jcollins-axway) [inline .github/workflows/build-apigee-discovery-agent-docker.yml:23]: extra quote at end?

File: .github/workflows/build-apigee-traceability-agent-docker.yml [hunk 3]
```diff
@@ -20,6 +20,7 @@ jobs:
         working-directory: traceability
         run: |
           git branch --show-current > commit_id
+          echo "${{ github.ref }}" > version"
```
Comment (jcollins-axway) [inline .github/workflows/build-apigee-traceability-agent-docker.yml:23]: extra quote at end?

**Review by jcollins-axway — CHANGES_REQUESTED** [review]
changes notes as comments

File: .github/workflows/build-apigee-discovery-agent-docker.yml (hunk 2 above)
Comment (dfeldick) [reply .github/workflows/build-apigee-discovery-agent-docker.yml:23]: fixed

File: .github/workflows/build-apigee-discovery-agent-docker.yml (hunk 1 above)
Comment (dfeldick) [reply .github/workflows/build-apigee-discovery-agent-docker.yml:19]: fixed

File: .github/workflows/build-apigee-traceability-agent-docker.yml (hunk 3 above)
Comment (dfeldick) [reply .github/workflows/build-apigee-traceability-agent-docker.yml]: fixed

**Pull request merged**